packages/agoric-cli/src/main.js

```diff
@@ -51,8 +51,6 @@ const main = async (progname, rawArgs, powers) => {
     );
   }
 
-  program.storeOptionsAsProperties(false);
```

Comment:
Okay, by reading this file's changes and studying https://www.npmjs.com/package/commander, this looks good.